microcosm_flask/audit.py

```diff
@@ -183,7 +185,10 @@ def log(self, logger):
                 logger.warning(dct)
         else:
             # usually log at INFO; a raised exception can be an error or expected behavior (e.g. 404)
-            logger.info(self.to_dict())
+            if self.options.log_as_debug:
```

Comment:
Since the main path (default?) is probably going to be to log this at info, this could be inverted (you'll eat the branch misprediction every time otherwise).

Reply:
Might be a huge premature optimization.

Reply:
Not a big deal, besides optimization, I agree our brains expect the `if` to be the common case.